src/View.ts

```diff
@@ -646,11 +646,13 @@ export class ViewWrapper extends EventHandler {
       });
 
       const groupsArray = Array.from(groups);
-      const categoryOrder = getPlugin('chooserConfig', 'chooser_header_order');
+
+      const orderDescs = listPlugins('chooserConfig').map((desc) => desc.order);
 
       let sortedGroups = null;
-      if(categoryOrder) {
-        sortedGroups = groupsArray.sort((a, b) => categoryOrder.order[a[0]] - categoryOrder.order[b[0]]);
+      if(orderDescs.length) {
+        const categoryOrder = Object.assign({}, ...orderDescs);
+        sortedGroups = groupsArray.sort((a, b) => categoryOrder[a[0]] - categoryOrder[b[0]]);
```

Comment:
what if an invalid group is chosen because of an configuration order? i.e, `categoryOrder[a[0]]` is undefined?